src/main/java/com/unosquare/cvgenerator/service/GreetingServiceImpl.java

```diff
@@ -1,31 +1,28 @@
 package com.unosquare.cvgenerator.service;
 
 import com.unosquare.cvgenerator.dao.HelloRepository;
-import com.unosquare.cvgenerator.model.dto.GreetingModelView;
+import com.unosquare.cvgenerator.model.dto.GreetingDTO;
 import com.unosquare.cvgenerator.model.entity.Greeting;
 import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
 import org.modelmapper.ModelMapper;
 import org.springframework.stereotype.Service;
 
 import java.util.Optional;
 
-@Slf4j
 @Service
 @RequiredArgsConstructor
 public class GreetingServiceImpl implements GreetingService {
 
     private final HelloRepository helloRepository;
+    private final ModelMapper modelMapper;
 
     @Override
-    public GreetingModelView findById(int id) {
+    public GreetingDTO findById(int id) {
         Optional<Greeting> result = helloRepository.findById(id);
         if (!result.isPresent()) {
             throw new RuntimeException(String.format("Cannot find object with id %s", id));
```

Comment:
Future change: Use more specific exception (or create custom exception) to better indicate "Not found"